Sources/Orbit/Support/Components/TileBorder.swift

```diff
@@ -97,17 +105,18 @@ public struct TileBorderModifier: ViewModifier {
     }
 
     var showOuterBorder: Bool {
-        style == .iOS
+        switch style {
+            case .iOS, .plain:      return true
+            case .none, .default:   return false
+        }
     }
 }
 
 public extension View {
 
-    /// Decorates content with a ``Tile`` appearance using specified style.
-    ///
-    /// - Parameter style: The style to apply. If style is nil, the view doesn’t get decorated.
+    /// Decorates content with a ``Tile`` or ``Card`` appearance using specified style.
```

Comment:
If it is truly only meant for `Tile` and `Card`, shouldn't there be extensions for only those types? 🤔

Reply:
It is more like decorates with Tile/Card-like appearance. Anyway, this modifier will be replaced in future with more composable modifiers (separate selectable border).

Reply:
Updated